cfme/tests/control/__init__.py

```diff
@@ -0,0 +1,39 @@
+from cfme.web_ui import flash, toolbar
+from utils import conf
+from utils.log import logger
+from utils.wait import wait_for
+
+vddk_url_map = {
+    "5.5": conf.cfme_data.get("basic_info", {}).get("vddk_url").get("v5_5"),
```

Comment:
The 2nd `get()` here could return None, resulting in an AttributeError on the 3rd `.get()`.

Consider turning this into a fixture instead, and skipping tests when these values aren't found.